phpunit/class-wp-theme-json-test.php

```diff
@@ -1702,6 +1702,34 @@ public function data_get_styles_for_block_with_style_variations() {
 		);
 	}
 
+	public function test_block_style_variations() {
+		wp_set_current_user( static::$administrator_id );
+
+		$expected = array(
+			'version' => WP_Theme_JSON_Gutenberg::LATEST_SCHEMA,
+			'styles'  => array(
+				'blocks' => array(
+					'core/button' => array(
+						'color'      => array(
+							'background' => 'blue',
+						),
+						'variations' => array(
+							'outline' => array(
+								'color' => array(
+									'background' => 'purple',
```

Comment:
Not a blocker, but to be extra thorough about the test, would it be worth adding an invalid property to the `outline` array, to test that it correctly gets removed by the call to `remove_insecure_styles`?

Reply:
Cheers @andrewserong I've added another test for that